docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1c.md

```diff
@@ -0,0 +1,71 @@
+"average" is not particularly well defined in this exception. 
+for the peak W/sf calculation I've assumed:
+	1. the zone being tested is not included
+ 	2. the average: (each space coincident peak W) / total_area
+
+**Inputs:** 
+- **B_RMD**
+- **zone_id**
+- **starting_system_type** - this is the expected system type that was selected prior to checking G3_1_1c exception
+
+**Returns:**  
+- **result**: an enum - either YES or NO
+
+**Function Call:**
+- **get_baseline_system_types**
+- **get_zone_peak_internal_load**
+- **get_list_hvac_systems_associated_with_zone**
+- **get_hvac_zone_list_w_area**
+- **find_zone**
+- **get_zone_eflh**
+- **zones_with_computer_room_dict_x**
+
+## Logic:
+- set the result variable to "No" - only a positive test can give it a different value: `result = NO`
+
+- first check if the system type is one of the following:
+`eligible_primary_system_types = ["SYS-5","SYS-5b","Sys-6","Sys-6b","Sys-7","Sys-7a","Sys-7b","Sys-7c","Sys-8","Sys-8a","Sys-8b","Sys-8c","Sys-8d","Sys-9","Sys-9b","Sys-10","Sys-11.1","Sys-11.1a","Sys-11b","Sys-11c","Sys-12","Sys-12a","Sys-12b","Sys-12c","Sys-13","Sys-13a"]`
+- check if the starting_system_type is in the list of eligible system types: `if starting_system_type in eligible_primary_system_types:`
+	- get the list of all systems and the zones they serve in the RMD by calling get_hvac_zone_list_w_area: `hvac_zone_list_w_area = get_hvac_zone_list_w_area(B_RMD)`
+	- loop through each system in eligible_systems_in_zone: `for eligible_system in eligible_systems_in_zone:`
+		- create a variable to hold the sum total of all peak loads in the system:
+		`system_total_peak_load = 0`
+		- create a variable to hold the sum total of the system area:
+		`system_total_area = hvac_zone_list_w_area[eligible_system.id]["TOTAL_AREA"]`
+		- create a list of the EFLH's for each zone `eflh = []`
+		- get the list of zones associated with this system: `zones_associated_with_system = hvac_zone_list_w_area[eligible_system.id]["ZONE_LIST"]`
+		- loop through the zones in the list: `for zone_id in zones_associated_with_system:`
+			- find the zone using the find_zone function: `zone = find_zone(zone_id)`
+				- check that zone_b is not the zone being investigated:
+				`if zone_b != zone:`
+					- add the zone internal loads to the system_total_peaks:
+					`a = get_zone_peak_internal_load(zone_b)`
+					`system_total_peak += a[0]`
+					- get the zone eflh by using the get_zone_eflh function.  This returns eflh for the year, so we need to convert to weeks by dividing by 52.1428: `zone_eflh = get_zone_eflh(zone)/52.1428`
+					- add the eflh to the eflh list.  We are multiplying the schedule by the zone area so that later we can divide the total by the total area.  Otherwise small zones with very different schedules have too great an impact
+					`eflh.append(zone_eflh * a[1])`
+		- now calculate the average internal load: `avg_internal_load = system_total_peak / system_total_area`
```

Comment:
Stylist comment: maybe can change the name to internal_load_per_floor_area to document this variable?

Reply:
changed